ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/main.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QGuiApplication>
+#include <QQuickView>
+#include <QCommandLineParser>
+#include <QDir>
+#include <QQmlEngine>
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+#include "ClipGeometry.h"
+
+#define STRINGIZE(x) #x
+#define QUOTE(x) STRINGIZE(x)
+
+int main(int argc, char *argv[])
+{
+
+  QGuiApplication app(argc, argv);
```

Comment:
this is missing the app.setApplicationName. You can manually add it in. If you get a fresh pull of the qt-sdk repo, you can get templates that have this addition built into it. Let me know if you have questions on that